### PR TITLE
Issu#136 region

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

**Comment by dnadeau4** [comment]
travis is failing due to "Ruby"  will try to rebuild when Travis fix it.

**Comment by dnadeau4** [comment]
We can no longer use the "... since 0"  as unit since year 0 is not CF-1 compliant.   I changed all tests and test_file `th_yr.nc` to reflect this.  This is a `python 3` issue.

**Comment by dnadeau4** [comment]
I also warned the users about the select_region and created a tests for ticket #22

**Comment by dnadeau4** [comment]
#136

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,14 @@
 os:
-    - linux
+  - linux
+
+language: python
+
+python:
```
Comment (doutriaux1) [inline .travis.yml]: @dnadeau4 why do we need this? Does it run the test twice now?

Comment (dnadeau4) [reply .travis.yml]: I removed python 3.6 since we already do both version in the `conda env`.

Comment (dnadeau4) [reply .travis.yml]: @doutriaux1 Can you merge it?

**Pull request merged**